doc/api_review/core/ksLookupByName.md

```diff
@@ -88,11 +88,13 @@
 
 - [x] Function code is fully covered by tests
 - [ ] All possible error states are covered by tests  
-       - [ ] `name == NULL`
+       - [x] `name == NULL`
 - [ ] All possible enum values are covered by tests  
-       - [ ] `KDB_O_DEL`
+       - [ ] `KDB_O_DEL`  
```

Comment:
Does `KDB_O_DEL` even make sense for `ksLookupByName()` ? Im getting Segfaults which would make sense, because the internally created Key would get freed 2 times.

Reply:
The combination is used src/plugins/list/list.c but is `KDB_O_DEL` even a good idea at all? It seems to me like a quite dangerous "convenience option". `KDB_O_DEL` is not used so much, so maybe simply getting rid of it is the best idea.

The list plugin gets obsolete with #3693

KDB_O_CREATE probably also needs to be removed, at least for cascading keys, see #3742

Reply:
Okay, I think its better to move forward with this PR and create a new Issue / PR for it